.coveragerc

```diff
@@ -0,0 +1,51 @@
+[run]
+parallel=True
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+    def __repr__
+
+    # Don't complain about missing debug-only code:
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if False:
+    if __name__ == .__main__.:
+
+    # Not counting some extra lines which don't need to be counted.
+    print
+    def setUp
+    def suite
+
+ignore_errors = True
+
+omit =
```

Comment:
We should also omit **aio_compat.py** and **typing_inspect.py**. These two files are imported from thirdparty repository. Let follow the steps to move these two files into the **\_thirdparty** folder.

1. Create a new folder, named **\_thirdparty**, moving these two files into it
2. Change the import statements where it references these two files
3. Change the setup.cfg
    - `mypy-azure_functions_worker.aio_compat => mypy-azure_functions_worker._thirdparty.aio_compat`
    - `mypy-azure_functions_worker.typing_inspect => `mypy-azure_functions_worker._thirdparty.typing_inspect`
4. Add a new line here `azure_functions_worker/_thirdparty/*` to omit any thirdparty files
5. Add a new line in `setup.py -> packages=[ -> azure_functions_worker._thirdparty` to bundle the files into the existing module.